knowledge-base/azure/k8s-deploy/action-security.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Deploy to Kubernetes cluster'
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
+    contents: read
+    contents-reason: to query workflow path
+    issues: write
+    issues-reason: to set issues as idle
```

Comment:
@arjundashrath can you please add a link to the reason, where it has a reason the contents-read and others are needed. That makes it easy for me to verify this. You can add it as a comment. See [this](https://github.com/step-security/secure-workflows/blob/8e3478a5c48e4e82ab08272884d0051785095c71/knowledge-base/Azure/static-web-apps-deploy/action-security.yml#L4) for example.

Reply:
https://github.com/Azure/k8s-deploy/blob/ac49626466364585521685ccaad17e499a8263a8/src/utilities/utility.ts#L88

https://github.com/Azure/k8s-deploy/blob/acd12a47055b679be711e4937c41f1755316cfae/.github/workflows/defaultLabels.yml#L20
the workflows contain token for Issue/PR management, I wonder if it takes it from the action input?